compiler/control/OMROptions.cpp

```diff
@@ -1376,7 +1378,7 @@ OMR::Options::getNumericValue(const char *& option)
    while (pendingOperation)
       {
       int64_t current = 0;
-      while (isdigit(*option))
+      while (__isdigit_a(*option))
```

Comment:
I don't think it's reasonable to define `_AE_BIMODAL` on all platforms, nor is it reasonable to assume that `__isdigit_a()` exists on all platforms.

I suggest we should introduce helper macros (e.g. `omr_isdigit()`) which would be suitably defined for each platform and uses of `isdigit()` would be replaced by uses of `omr_isdigit()`. The same approach should be applied for `toupper()`, `tolower()`, etc.

Reply:
Sounds good, I can define that in `util/a2e/headers/ctype.h` I guess.
Is there a `__tolower_a()` and `__toupper_a()` similarly available as well? Not able to find documentation around it.

Reply:
No. `omr_isdigit()` would be defined in some OMR header file (perhaps after including `ctype.h`).